main.tf

```diff
@@ -42,16 +42,14 @@ locals {
     "ecr:GetDownloadUrlForLayer",
   ]
 
-  ecr_push_only_actions = [
+  ecr_push_actions = [
     "ecr:CompleteLayerUpload",
     "ecr:InitiateLayerUpload",
     "ecr:ListImages",
     "ecr:PutImage",
     "ecr:UploadLayerPart",
   ]
 
-  ecr_push_actions = concat(local.ecr_push_only_actions, local.ecr_pull_actions)
-
```

Comment:
this is not a cleanup this is a breaking change in behavior.

Reply:
resolved in  89ff85b